tests/unit/igraph_layout_umap.c

```diff
@@ -76,7 +74,7 @@ int main() {
 #endif
 
     igraph_rng_seed(igraph_rng_default(), 42);
-    igraph_small(&graph, 4, IGRAPH_UNDIRECTED, // 4?
+    igraph_small(&graph, 12, IGRAPH_UNDIRECTED,
```

Comment:
Just noting that in `igraph_small()` you may choose to provide `0` as the vertex count and let the function figure out the correct value.

Reply:
Yes, I had no clue, but that's also apparently what happened before. Learned something new 😃